### PR TITLE
[RTC-79] Setup docker CD

### PR DESCRIPTION
**Pull request by LVala**

### CODE REVIEW

**Comment by LVala** [comment]
I created docker image, which worked on [this branch with basic REST API](https://github.com/jellyfish-dev/jellyfish/tree/RTC-77-basic-rest-api), but to truly test if everything is alright we need a working application (i.e. be able to send media to hls endpoint), so for the time being this pr can wait.

**Review by Rados13 — APPROVED** [review]
What do you think about setup [elixir test containers](https://github.com/dallagi/excontainers), this way we could probably check whether docker is building correctly, besides that PR looks good.

**Review by Rados13 — APPROVED** [review]

File: .env.sample [hunk 1]
```diff
@@ -1,3 +1,13 @@
...
+SECRET_KEY_BASE=super-secret-key
+
+# true, if WebRTC peers are used
+WEBRTC_USED=true
+
+# hostname used to generate URLs throught the server
 VIRTUAL_HOST=localhost
+
+# TURN default configuration
 INTEGRATED_TURN_IP=127.0.0.1
```
Comment (Rados13) [inline .env.sample]: What do you think about adding information that loopback should be used only in local development?

**Review by Rados13 — APPROVED** [review]

**Review by shuntrho — APPROVED** [review]

File: .env.sample [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+# true, if WebRTC peers are used
+WEBRTC_USED=true
+
+# hostname used to generate URLs throught the server
+VIRTUAL_HOST=localhost
+
+# TURN default configuration
+# note: loopback address as INTEGRATED_TURN_IP cannot be used inside a Docker container
+INTEGRATED_TURN_IP=127.0.0.1
```
Comment (mickel8) [inline .env.sample:12]:
```suggestion
INTEGRATED_TURN_IP=192.168.0.1
```

**Review by mickel8 — APPROVED** [review]

**Pull request merged**